test/TestHelper.cpp

```diff
@@ -547,26 +545,21 @@ bytes importData(json_spirit::mObject const& _o)
 
 std::string compileLLL(std::string const& _code)
 {
-	FILE *fp;
-	char path[1035];
-	FILE *ft = fopen("temp.sol", "w");
-	if (ft == NULL)
-		cerr << "Error creating temp file for lllc";
-	else
-		fputs(_code.c_str(), ft);
-	fclose(ft);
-
-	fp = popen("../../solidity/lllc/lllc temp.sol", "r");
+	char input[1024];
+	boost::filesystem::path path = boost::filesystem::unique_path();
+	std::string filename = path.string() + "/code.sol";
```

Comment:
`unique_path` returns a file name, not a directory name, so please just use `path.string()`.